tests/test_spherePoint.py

```diff
@@ -147,23 +160,29 @@ def testInitNArgFail(self):
             SpherePoint(42)
         with self.assertRaises(TypeError):
             SpherePoint("ICRS", 34.0, -56.0)
+        with self.assertRaises(TypeError):
+            SpherePoint(34.0, -56.0)  # missing units
 
     def testGetLongitudeValue(self):
         """Test if getLongitude() returns the expected value.
         """
         for lon, lat in self._dataset:
-            point = SpherePoint(lon, lat)
-            self.assertIsInstance(point.getLongitude(), afwGeom.Angle)
-            # Behavior for non-finite points is undefined; depends on internal
-            # data representation
-            if point.isFinite():
-                self.assertGreaterEqual(point.getLongitude().asDegrees(), 0.0)
-                self.assertLess(point.getLongitude().asDegrees(), 360.0)
-
-                # Longitude not guaranteed to match input at pole
-                if not point.atPole():
-                    # assertAnglesAlmostEqual handles angle wrapping internally
-                    self.assertAnglesAlmostEqual(lon, point.getLongitude())
+            for point in (
+                SpherePoint(lon, lat),
+                SpherePoint(lon.asDegrees(), lat.asDegrees(), degrees),
+                SpherePoint(lon.asRadians(), lat.asRadians(), radians),
+            ):
```

Comment:
Why not just use `self.pointSet`?

Reply:
I do this in two places because in those places the test needs the input longitude or latitude and pointSet does not provide them. I considered a property pointLonLatSet but I felt it was excessive clutter for just two tests.

Reply:
Good point. A comment explaining that might be helpful, though.